Showroom/Sidebar/styles.scss

```diff
@@ -1,8 +1,11 @@
+@import '../../css/settings';
 @import '../settings.scss';
+@import '../../css/mixins/index';
 
 .sidebar {
   -webkit-font-smoothing: antialiased;
```

Comment:
I've seen this a few times now I think.
Wouldn't this be more of a base style `html,body` rule than on certain elements only ?